client/src/rpc_client.rs

```diff
@@ -474,6 +474,13 @@ impl RpcClient {
         self.get_epoch_info_with_commitment(CommitmentConfig::default())
     }
 
+    pub fn get_stake_activation(&self, pubkey: &Pubkey) -> ClientResult<RpcStakeActivation> {
```

Comment:
I think adding new `fn`s is ok in terms of compatibility even with a patch release.